src/components/RadioPanel/RadioPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode, Ref, forwardRef } from 'react'
```

Comment:
What do you think of adding this radio panel to the radio folder? Similar to the `RadioWithLabel` or `RadioGroup` (that we export), it's a implemenation of the `Radio` component